features/registerprotocolhandler.yml

```
@@ -0,0 +1,6 @@
+name: registerProtocolHandler
+description: "The `navigator.registerProtocolHandler()' method lets websites register their ability to open or handle particular address schemes (also known as protocols). For example, this API lets webmail sites open `mailto:` URLs, or VoIP sites open `tel:` URLs."
```

I cannot remember the last time I heard the word "webmail." Also, some edits for brevity:

```suggestion
description: "The `navigator.registerProtocolHandler()' method declares a site's ability to handle an address scheme (also known as a protocol). For example, an email site can register to open `mailto:` URLs or a VoIP site to open `tel:` URLs."
```